backend/de.metas.business/src/main/java/org/adempiere/mm/attributes/api/impl/AttributeDAO.java

```diff
@@ -256,10 +268,14 @@ private static Attribute toAttribute(final I_M_Attribute record)
 		final ITranslatableString displayName = InterfaceWrapperHelper.getModelTranslationMap(record)
 				.getColumnTrl(I_M_Attribute.COLUMNNAME_Name, record.getName());
 
+		final ITranslatableString description = InterfaceWrapperHelper.getModelTranslationMap(record)
```

Comment:
pls extract `InterfaceWrapperHelper.getModelTranslationMap(record)` to local variable
(including the one above)

Reply:
extracted.